habitat-lab/habitat/config/habitat/task/rearrange/multi_agent_tidy_house_base_spot.yaml

```diff
@@ -39,8 +39,11 @@ constraint_violation_drops_object: True
 task_spec: multi_agent_tidy_house_fp
 pddl_domain_def: fp
 actions:
-  # Actions are defined per agent. Unlike the sensors, actions are not
-  # automatically duplicated per agent.
+  agent_0_base_velocity:
+    agent_index: 0
+    lin_speed: 40.0
```

Comment:
Why are setting agent_0_base_velocity? Sorry a little confused, I thought we were only using oracle nav? Is this still correct?